README.md

```diff
@@ -1,2 +1,6 @@
 # SRK
 Stoned Fairy You
+
+# SFY
+
+Stoned Rocket Kid
```

Comment:
Why do you stoned the rocket kid?

Reply:
Yummy isn't right

Reply:
Could you send me a snippet, I could copy and paste?